lib/committee/schema_validator/open_api_3.rb

```diff
@@ -1,5 +1,6 @@
 class Committee::SchemaValidator
-  class OpenAPI3
+    class OpenAPI3
```

Comment:
[nits] It looks indentation is mistaken.

Reply:
fixed!